src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -8,12 +8,12 @@
 import frc.robot.constants.RobotConstants.IntakeConstants;
 
 public class Intake extends SubsystemBase {
-  private final CANSparkMax m_rollerMotor; // Intake roller motor
+  private final CANSparkMax m_intakeRollerMotor; // Intake roller motor
   private final DigitalInput m_linebreak;
 
-  // Constructs intake and initializes motor
+  /** Creates a new ExampleSubsystem. */
```

Comment:
Probably best to remove this comment

Reply:
yep we should but as long as its commented were keeping it